indigo/bulk_creator.py

```diff
@@ -409,25 +409,32 @@ def link_commencement(self, work, info):
         commencement.save()
 
     def link_repeal(self, work, info):
-        # if the work is `repealed_by` something, try linking it
-        # make a task if this fails
-        # (either because the work isn't found or because the repeal date isn't right,
-        # which could be because it doesn't exist or because it's in the wrong format)
+        # if the work is `repealed_by` something, try linking it or make the relevant task
         repealing_work = self.find_work_by_title(info['repealed_by'])
+
         if not repealing_work:
-            return self.create_task(work, info, task_type='link-repeal')
+            # an exact match on the given title wasn't found
+            self.create_task(work, info, task_type='no-repeal-title-match')
 
-        repeal_date = repealing_work.commencement_date
-        if not repeal_date:
-            return self.create_task(work, info, task_type='link-repeal')
+        elif work.repealed_by and work.repealed_by.id != repealing_work.id:
```

Comment:
you should be able to use `elif work.repealed_by and work.repealed_by != repealing_work`

Reply:
cool, thanks!